API/models/comment-model.js

```diff
@@ -8,3 +8,14 @@ exports.fetchComments = (article_id) => {
     [article_id]
   );
 };
+
+exports.saveComment = (article_id, comment) => {
+  const username = comment.username;
+  const body = comment.body;
```

Comment:
Best to either consistently use destructuring throughout the project or write longhand, rather than a mix of the two